docs/API/models.md

```
@@ -0,0 +1,92 @@
+# Models API
+
+## List Available Models
+
+Returns the consolidated model catalog for all providers that have credentials configured in `agent.yml`.
+
+### Request
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)